docs/docs/API-Reference/api-openai-responses.mdx

```diff
@@ -0,0 +1,303 @@
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id | string | Unique response identifier |
+| object | string | Always "response" |
+| created_at | int | Unix timestamp of response creation |
+| status | string | Response status: "completed", "in_progress", or "failed" |
+| error | dict | Error details (if any) |
+| incomplete_details | dict | Incomplete response details (if any) |
+| instructions | string | Response instructions (if any) |
+| max_output_tokens | int | Maximum output tokens (if any) |
+| model | string | The flow ID that was executed |
+| output | list[dict] | Array of output items (messages, tool calls, etc.) |
+| parallel_tool_calls | boolean | Whether parallel tool calls are enabled |
+| previous_response_id | string | ID of previous response if continuing conversation |
+| reasoning | dict | Reasoning information with effort and summary |
+| store | boolean | Whether response is stored |
+| temperature | float | Temperature setting |
+| text | dict | Text format configuration |
+| tool_choice | string | Tool choice setting |
+| tools | list[dict] | Available tools |
+| top_p | float | Top-p setting |
+| truncation | string | Truncation setting |
+| usage | dict | Usage statistics (if any) |
+| user | string | User identifier (if any) |
+| metadata | dict | Additional metadata |
+
+## Example streaming request
+
+```curl
+curl -X POST \
+  "$LANGFLOW_SERVER_URL/api/v1/responses" \
+  -H "x-api-key: $LANGFLOW_API_KEY" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "model": "your-flow-id",
+    "input": "Tell me a story about a robot",
+    "stream": true
+  }'
+```
+
+<details>
+<summary>Response</summary>
+
+```json
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123456, "model": "your-flow-id", "delta": {"content": "Once"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123457, "model": "your-flow-id", "delta": {"content": " upon"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123458, "model": "your-flow-id", "delta": {"content": " a"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123459, "model": "your-flow-id", "delta": {"content": " time"}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123460, "model": "your-flow-id", "delta": {"content": "..."}}
+{"id": "response-uuid", "object": "response.chunk", "created": 1703123461, "model": "your-flow-id", "delta": {}, "status": "completed"}
+```
+
+</details>
+### Streaming response body
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id | string | Unique response identifier |
+| object | string | Always "response.chunk" |
+| created | int | Unix timestamp of chunk creation |
+| model | string | The flow ID that was executed |
+| delta | dict | The new content chunk |
+| status | string | Response status: "completed", "in_progress", or "failed" (optional) |
+
+## Pass global variables to your flows in headers
+
```

Comment:
Does this mean you are setting values for global variables that are named in the flow? It will override any value set in Langflow itself?

I just want to understand clearly: You can't arbitrarily declare a global variable that isn't declared in the flow or related to flow execution?

Reply:
Explained below